scripts/ubuntu20_04_build_setup.sh

```diff
@@ -0,0 +1,23 @@
+
+echo "Install dependencies"
+apt update
+apt install -y git build-essential python3.8 python3-pip pkg-config mesa-common-dev libglu1-mesa-dev wget libpcsclite-dev libpcre3-dev libssl-dev libpulse-mainloop-glib0 libxkbcommon-x11-dev extra-cmake-modules cmake
+
+echo "Install QT"
+pip install -U pip
+pip install aqtinstall
+aqt install-qt linux desktop 5.14.2 gcc_64 -m qtwebengine qtlottie
```

Comment:
Does this work to install all dependencies? The last time I checked the docs, it said to use the online installer